languages/go/exercises/.docs/debug.md

```diff
@@ -0,0 +1,11 @@
+# Debug
```

Comment:
The location for this file should change to `exercises/shared/.docs/debug.md`. See #54.